packages/interaction-kit/src/components/action-row.ts

```diff
@@ -15,9 +15,8 @@ export default class ActionRow implements SerializableComponent {
 		this.setComponents(...components);
 	}
 
-	// TODO: Is there a cleaner form for this?
 	get id() {
-		return "";
+		return undefined;
```

Comment:
When explicitly declaring something as non-existent, you should return `null`.
```suggestion
		return null;
```

Not sure if this breaks anything somewhere else in the code?

Reply:
It doesn't adhere to the Discord API types, which is why I left it undefined - I can add a comment though